docs/cli-reference.adoc

```diff
@@ -625,22 +627,14 @@ _________________
   odo preference unset  Timeout
 ----
 
-Modifies odo specific configuration settings within the global preference file.
-
-[NOTE]
-====
-By default the path to the global preference file is `~/.odo/preferece.yaml`, it is stored in in the environment variable `GLOBALODOCONFIG`. You can set up a custom path by setting the value of the environment variable to a new preference path, for example `GLOBALODOCONFIG="new_path/preference.yaml"`
```

Comment:
Why was this removed?

Reply:
This was done by the script, I can add it back, but it also means script needs to be updated

Reply:
Added back, but er should probably fix the scripts/template accordingly